bundle/src/main/java/org/cru/importer/GiveDataImportServlet.java

```diff
@@ -123,6 +124,14 @@ private boolean validateParams(SlingHttpServletRequest request, ParametersCollec
 				} else {
 					parametersCollector.setAcceptFilesPattern(acceptFilesPattern);
 				}
+	            Resource globalConfigs = getGlobalConfig(request, properties);
+	            if (globalConfigs == null) {
+	                resultsCollector.addError("globalConfigs configuration not found at configpath or node not exists at JCR.");
+	                isValid = false;
+	            } else {
+	                ValueMap globalProperties = globalConfigs.adaptTo(ValueMap.class);
+	                parametersCollector.setSanitizationMap(PropertiesUtil.toMap(globalProperties.get("sanitizationMap",String[].class), new String[]{}));
+	            }
```

Comment:
@mattdrees ok.  we can fix along with the staffweb implementation which we're starting today; it is based on this very same code.